### PR TITLE
Adding s2i builder for jekyll

### PR DESCRIPTION
**Pull request by etsauer**
#### What is this PR About?
Adds a builder image for generating jekyll sites

#### How do we test this?
Follow README

cc: @redhat-cop/containerize-it @redhat-cop/cant-contain-this

### CODE REVIEW

File: s2i-jekyll/s2i/run [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+SOURCE_DIR=/tmp/src
+
+if [ ! -d "$SOURCE_DIR" ]; then
+   echo "Error: Source volume not mounted or available"
+   exit 1
+fi
+
+cd "$SOURCE_DIR"
+
+bundle exec jekyll serve --no-watch --host=0.0.0.0
```
Comment (ghyde) [inline s2i-jekyll/s2i/run]: The jekyll server is designed for development purposes, not for actually serving websites. For production, I would use a web server (Nginx, Apache, etc.) to serve the generated HTML.

Comment (etsauer) [inline s2i-jekyll/s2i/run]: @ghyde agreed. the purpose of this image is mainly to build. the s2i/run script is just for testing/validation.

File: s2i-jekyll/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# S2I Image for Building Jekyll Sites
+
+This image builds sites using [Jekyll](https://jekyllrb.com/).
+
+## Quickstart
+
+```
+oc new-build https://github.com/etsauer/containers-quickstarts#s2i-jekyll --strategy=docker --context-dir=s2i-jekyll --name='jekyll-builder'
```
Comment (ghyde) [inline s2i-jekyll/README.md]: Should this point to `redhat-cop` instead of your personal repo?

File: s2i-jekyll/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+oc new-app https://github.com/redhat-cop/openshift-playbooks --image-stream=jekyll-builder
+```
```
Comment (sabre1041) [inline s2i-jekyll/README.md]: Need to add a step to expose the service to create a route

```
oc expose svc openshift-playbooks
```

**Comment by etsauer** [comment]
@sabre1041 @ghyde enhanced the reamde doc based on your feedback.

**Comment by sabre1041** [comment]
@etsauer LGTM

@ghyde good from your side?

**Pull request merged**